README.md

```diff
@@ -72,3 +72,20 @@ To remove the virtual environment
 make rm_env
 ```
 
+### Troubleshooting
+Ubuntu 20.04 users might experience issues when creating the environment (running `make env`) due to an older version of `setuptools` installed on the virtual environment.
+To workaround this issue, update the version of `pipenv` to `2020.11.4` or newer.
+```
+python -m pip uninstall pipenv
+pip install --user pipenv
```

Comment:
can't pipenv be upgraded with `pip install --upgrade --user pipenv` instead?

Reply:
Yes and actually looks more _natural_ to update using that instead of removing it and installing it again. 
Thanks.